src/Airship/Types.hs

```diff
@@ -140,6 +147,7 @@ instance MonadBaseControl b m => MonadBaseControl b (Webmachine m) where
 -- | A convenience synonym that writes the @Monad@ type constraint for you.
 type Handler m a = Monad m => Webmachine m a
```

Comment:
https://github.com/helium/airship/pull/98

On Tue, Mar 29, 2016 at 9:13 AM, Reid Draper <notifications@github.com>
wrote:

> In src/Airship/Types.hs
> <https://github.com/helium/airship/pull/94#discussion_r57642647>:
>
> > @@ -140,6 +147,7 @@ instance MonadBaseControl b m => MonadBaseControl b (Webmachine m) where
> >  -- | A convenience synonym that writes the @Monad@ type constraint for you.
> >  type Handler m a = Monad m => Webmachine m a
>
> I think we removed our use of it, too. Definitely happy to get rid of it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/helium/airship/pull/94/files/f9ade37a41c7525f6be30ea2cfc32c71c5929578#r57642647>
>